test/pages/register-a-beacon/additional-beacon-use.test.tsx

```diff
@@ -70,6 +73,15 @@ describe("AdditionalBeaconUse page", () => {
     expect(within(content).getByText(new RegExp(use2.purpose, "i")));
   });
 
+  it("given there are one or many uses, allows the user to add a use via a button", () => {
+    render(<AdditionalBeaconUse uses={[]} currentUseIndex={0} />);
```

Comment:
I might be missing something here (too hottt 🌞) but do we need to add some uses here to match the test description? Like the test above?

```suggestion
    const use1 = getMockUse();
    const use2 = {
      ...getMockUse(),
      environment: Environment.AVIATION,
      purpose: Purpose.COMMERCIAL,
      activity: Activity.GLIDER,
    };
render(<AdditionalBeaconUse uses={[use1, use2]} currentUseIndex={0} />);
```

Reply:
Agreed!  Thank you :-)